src/node.js

```diff
@@ -206,6 +207,13 @@ class Node {
             this.api.apiHost = addr.address
             this.api.apiPort = addr.port
 
+            if (gwmatch) {
+              this.gatewayAddr = gwmatch[2]
+              const addr = multiaddr(this.gatewayAddr).nodeAddress()
```

Comment:
I believe following the current convention as per api host and port is a better approach for consistency.

We can do what you propose in separate PR, bearing in mind it would be a public API change.